code/orders-api-rest/api/openapi-rest.yml

```
@@ -807,122 +801,179 @@ paths:
     get:
       tags:
-        - ViewHistory
-      summary: Get list of viewed products
-      description: The endpoint for getting all viewed products from the database.
-      operationId: getViewedProducts
-      security:
-        - bearerAuth: [ ]
+        - PasswordResetController
+      summary: Validate password reset token
+      description: |
+        Validates a token and returns its status. Used to enable the password 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)